docker-compose.yml

```diff
@@ -48,6 +48,7 @@ services:
       SECRET_KEY: ${SECRET_KEY}
       ANTHROPIC_API_KEY: ${ANTHROPIC_API_KEY}
       OPENAI_API_KEY: ${OPENAI_API_KEY}
+      GOOGLE_API_KEY: ${GOOGLE_API_KEY}
```

Comment:
this actually doesn't need to be here. It needs to be under the admin-worker, which is actually where the prompt happens.